grafana/resource_alerting_notification_policy_test.go

```diff
@@ -0,0 +1,97 @@
+package grafana
+
+import (
+	"fmt"
+	"testing"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccNotificationPolicy_basic(t *testing.T) {
+	CheckOSSTestsEnabled(t)
+	CheckOSSTestsSemver(t, ">=9.0.0")
```

Comment:
This depends on 9.1, will change once the container image is published.

Not changing it now so that the CI stays broken. That way nobody accidentally submits this too early.